chapters/fa/glossary/1.mdx

```diff
@@ -42,21 +50,27 @@
 | macOS                 | سیستم‌عامل مک     |
 | Distribution          | توزیع            |
 | Linux                 | لینوکس           |
-| Workload              | محاسبه، محاسبات          |
+| Workload              | محاسبه، محاسبات |
 | Package Manager       | پکیج‌منیجر        |
-| Command               | دستور یا فرمان            |
-| Feature, as for an app               | قابلیت‌           |
-| Development           | توسعه   |
+| Command               | دستور یا فرمان   |
+| Feature, as for an app | قابلیت‌          |
+| Development           | توسعه            |
+| Versioning            | نسخه‌بندی         |
+| Reproducibility       | قابلیت تکرارپذیری |
+| Git                   | گیت               |
+| GitHub                | گیت‌هاب            |
+| Repository            | مخزن              |
 | Dependency            | وابسته، وابستگی  |
+| Website               | وب‌سایت           |
 | Virtual Environment   | محیط مجازی       |
 | Terminal              | ترمینال          |
 | Incompatibility       | ناسازگاری        |
 | Self-Contained        | قائم به خود    یا      بدون وابستگی خارجی؟      |
 | Script                | اسکریپت‌          |
 | Folder | پوشه |
-| Neural Network | شبکه‌ی عصبی |
+| Neural Network | شبکه عصبی |
 | Text | نوشته |
-| Pipeline | خط تولید |
+| Pipeline | خط‌تولید |
```

Comment:
I don't think this should be one word. Check here https://fa.wikipedia.org/wiki/%D8%AE%D8%B7_%D8%AA%D9%88%D9%84%DB%8C%D8%AF

Reply:
Totally agree. I just modified it because of the half-space in the same title in _toctree.yml file. We should modify that file as well to remove the half-space then.

Reply:
> I don't think this should be one word. Check here https://fa.wikipedia.org/wiki/%D8%AE%D8%B7_%D8%AA%D9%88%D9%84%DB%8C%D8%AF

Totally agree. I just modified it because of the half-space in the same title in _toctree.yml file. We should modify that file as well to remove the half-space then.